system/lib/wasmfs/syscalls.cpp

```diff
@@ -1521,8 +1521,7 @@ int __syscall_fcntl64(int fd, int cmd, ...) {
     case F_SETLKW: {
       static_assert(F_SETLK == F_SETLK64);
       static_assert(F_SETLKW == F_SETLKW64);
-      // Always error for now, until we implement byte-range locks.
-      return -EACCES;
+      return 0; // Pretend that the locking is successful.
```

Comment:
```suggestion
      // Pretend that the locking is successful. These are process-level locks,
      // and Emscripten programs are a single process. If we supported linking a
      // filesystem between programs, we'd need to do more here.
      return 0;
```

Reply:
Done, plus I opened a bug to track this.